librb/src/epoll.c

```diff
@@ -36,7 +36,7 @@
 #include <fcntl.h>
 #include <sys/epoll.h>
 
-#if defined(HAVE_SIGNALFD) && (HAVE_SYS_SIGNALFD_H) && (USE_TIMER_CREATE) && (HAVE_SYS_UIO_H)
+#if defined(HAVE_SIGNALFD) && (HAVE_SYS_SIGNALFD_H) && (USE_TIMER_CREATE)
```

Comment:
Minor nitpick here (unrelated to your changes) but this should be

```C
#if defined(HAVE_SIGNALFD) && defined(HAVE_SYS_SIGNALFD_H) && defined(USE_TIMER_CREATE)
```

Reply:
Fixed!